sa/_db/migrations/20150904121736_CertificateStatusNotNullManyFields.sql

```diff
@@ -0,0 +1,23 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+ALTER TABLE `certificateStatus` MODIFY `subscriberApproved` tinyint(1) NOT NULL;
+ALTER TABLE `certificateStatus` MODIFY `status` varchar(255) NOT NULL;
+ALTER TABLE `certificateStatus` MODIFY `ocspLastUpdated` datetime NOT NULL;
+ALTER TABLE `certificateStatus` MODIFY `revokedDate` datetime NOT NULL;
```

Comment:
Unfortunately, the code is documented as saying that revokedData's "NULL" equivalent is the Unix epoch. CertificateStatus has a `time.Time` (not a `*time.Time`) to support that.

I'm not a fan but perhaps someone can comment on why we did that. 

If we don't have a good reason, it could be changed in my opinion.